docs_src/microservices/support/notifications/Ch-AlertsNotifications.md

```diff
@@ -35,19 +35,7 @@ channels include SMS message, e-mail, REST callback, AMQP, MQTT, and so
 on. Currently in EdgeX Foundry, e-mail and REST callback channels are
 provided.
 
-When Alerts and Notifications receive notifications from any interface,
-the notifications are passed to the Notifications Handler internally.
-The Notifications Handler persists the receiving notification first, and
-passes them to the Distribution Coordinator immediately if the
-notifications are critical (severity = "CRITICAL"). For normal
-notifications (severity = "NORMAL"), they wait for the Message Scheduler
-to process in batch.
-
-The Alerts and Notifications is scalable, can be expanded to add more
-severities and set up corresponding Message Schedulers to process them.
-For example, the Message Scheduler of normal severity notifications is
-triggered every two hours, and the minor severity notifications is
-triggered every 24 hours, at midnight each night.
```

Comment:
Was the corresponding functionality removed?

Reply:
Actually, I confirmed a gap in that the scheduling functionality as described (above) in the docs is currently _not_ implemented, so there was no need to remove. More details in the following comment where I was following up on feedback from @tsconn23: https://github.com/edgexfoundry/edgex-go/issues/2328#issuecomment-581905533